### ISSUE TITLE
5.7.0-1 breaks MySQL checking without tags

### ISSUE THREAD

**Issue by gtaylor** [DataDog/dd-agent#2331]
Debian Wheezy running the standard Datadog-provided apt packages. Downgrading to 5.6 fixes this.
```
8) | Running check mysql
2016-03-08 19:45:57 UTC | ERROR | dd.collector | checks.mysql(mysql.py:307) | error!
Traceback (most recent call last):
  File "/opt/datadog-agent/agent/checks.d/mysql.py", line 300, in check
    self._collect_metrics(host, db, tags, options, queries)
  File "/opt/datadog-agent/agent/checks.d/mysql.py", line 577, in _collect_metrics
    self._submit_metrics(metrics, results, tags)
  File "/opt/datadog-agent/agent/checks.d/mysql.py", line 598, in _submit_metrics
    metric_tags = list(tags)
TypeError: 'NoneType' object is not iterable
2016-03-08 19:45:57 UTC | ERROR | dd.collector | checks.mysql(__init__.py:763) | Check 'mysql' instance #0 failed
Traceback (most recent call last):
  File "/opt/datadog-agent/agent/checks/__init__.py", line 746, in run
    self.check(copy.deepcopy(instance))
  File "/opt/datadog-agent/agent/checks.d/mysql.py", line 308, in check
    raise e
TypeError: 'NoneType' object is not iterable
```
mysql.yaml:
```
init_config:

instances:
  - server: localhost
    user: datadog
    pass: PASSWORD-HERE
```
Adding a BS tag seems to make things happy again:
```
init_config:

instances:
  - server: localhost
    user: datadog
    pass: PASSWORD-HERE
    tags:
      - optional_tag1
```

**Comment by olivielpeau** [comment]
Hi @gtaylor, thanks for reporting this issue!

The problem has been reported in #2326 and fixed in #2329, we'll release a bugfix version of the agent this week that includes the fix.

Until then a workaround that doesn't require adding an actual tag is to use `tags: []` instead in the yaml file.

Thanks for your patience!

**Comment by gtaylor** [comment]
Whoops, I searched for the wrong thing when looking for an existing issue. Sorry about that!

Thanks for the workaround.
